tasks.py

```diff
@@ -652,7 +652,8 @@ def unittest(
         build_and_check_docs(context)
 
     append_arg = " --append" if append else ""
-    command = f"coverage run{append_arg} --module nautobot.core.cli --config=nautobot/core/tests/nautobot_config.py test {label}"
+    command = f"coverage run{append_arg} --module nautobot.core.cli test {label}"
+    command += " --config=nautobot/core/tests/nautobot_config.py"
```

Comment:
Is this change strictly for code formatting?

Reply:
Also re-ordering the parameters so that the above `TESTING` definition would correctly detect this as a test.